CHANGELOG.rst

```diff
@@ -1,6 +1,27 @@
 Change log
 ==========
 
+0.19.13 (2022-10-10)
+--------------------
+
+New features
+~~~~~~~~~~~~
+
+* **automation:** add Github Action automation server support
+* **out:** add Github Actions environment support
+* **config:** add path to configuration file in :doc:`configuration_support <configuration_support>`
+* **report:** add option to switch failures report to short form
+
+Bug fixes
+~~~~~~~~~
+
+* **config:** throw an error if project config is empty,
```

Comment:
There is no change in throwing errors. The difference is in error message for filters.